packages/ddl-shim/src/dermatology.ts

```diff
@@ -0,0 +1,229 @@
+    },
+    yAxisTypeTimePattern: {
+        name: "yAxisTypeTimePattern"
+    },
+    valueFormat: {
+        name: "tooltipValueFormat"
+    },
+    stacked: {
+        name: "yAxisStacked"
+    },
+    showYGrid: {
+        name: "yAxisGuideLines"
+    },
+    showXGrid: {
+        name: "xAxisGuideLines"
+    },
+    showValueLabel: {
+        name: "showValue"
+    },
+    low: {
+        name: "yAxisDomainLow"
+    },
+    high: {
+        name: "yAxisDomainHigh"
+    },
+    fillOpacity: {
+        name: "interpolateFillOpacity"
+    },
+    areaFillOpacity: {
+        name: "interpolateFillOpacity"
+    },
+    showToolbar: {
+        name: "titleVisible"
+    },
+    showCSV: {
+        name: "downloadButtonVisible"
+    }
+};
+
+function findKeyVal(object: any, key: any, val: any): any {
+    let value;
+
+    for (const k in object) {
+        if (k === key && object[k] === val) {
+            value = object;
+            break;
+        }
+        if (object[k] && typeof object[k] === "object") {
+            value = findKeyVal(object[k], key, val);
+            if (value !== undefined) {
+                break;
+            }
+        }
+    }
+    return value;
+}
+
+function apply_to_dataviews(ddl2: DDL2.Schema, dermObj: any) {
+
+    ddl2.dataviews.forEach(apply_to_dataview);
+
+    function apply_to_dataview(dv: any) {
+        const widgetId = dv.id;
+        const dermPanelObj: any = findKeyVal(dermObj, "__id", widgetId);
+        if (dermPanelObj) {
+            const dermPanelProps: any = dermPanelObj.__properties;
+            const dermWidgetObj: any = dermPanelObj.__properties.chart ? dermPanelObj.__properties.chart : dermPanelObj.__properties.widget;
+            const dermWidgetProps: any = dermWidgetObj.__properties;
+            apply_class_mapping(dermWidgetObj);
+            apply_panel_property_mapping(dermPanelProps, dermWidgetProps);
+            apply_widget_property_mapping(dermPanelProps, dermWidgetProps);
+            if (dv.visualization.properties.chartType) {
+                dv.visualization.properties.charttype = dv.visualization.properties.chartType;
+            }
+        } else {
+            console.warn(widgetId + " not found in dermObj");
+        }
+
+        function apply_class_mapping(dermWidgetObj: any) {
+            dv.visualization.__class = swap_with_supported_class(dermWidgetObj.__class);
+            dv.visualization.properties.__class = "marshaller_VizChartPanel";
+            if (!dv.visualization.properties.widget) dv.visualization.properties.widget = {};
+            dv.visualization.properties.widget.__class = dv.visualization.__class;
+
+            function swap_with_supported_class(_class: string): string {
+                return classMappings[_class] ? classMappings[_class] : _class;
+            }
+        }
+        function apply_panel_property_mapping(dermPanelProps: any, dermWidgetProps: any) {
+            dv.visualization.title = dermPanelProps.title || "";
```

Comment:
This line should be replaced with:
`if(dv.visualization.title === "")dv.visualization.title = dermPanelProps.title || "";`